tests/resources/carbon-metrics-gen.yaml

```diff
@@ -0,0 +1,108 @@
+              import time
+              import os
+              
+              # Configure logging
+              logging.basicConfig(
+                  level=logging.INFO,
+                  format='%(asctime)s - %(levelname)s - %(message)s',
+                  handlers=[logging.StreamHandler(sys.stdout)]
+              )
+              logger = logging.getLogger(__name__)
+              
+              logger.info("Starting Carbon metric generator...")
+              
+              # Wait for collector to be ready
+              RELEASE_NAME = os.environ.get('RELEASE_NAME', 'logzio-k8s-telemetry')
+              NAMESPACE = os.environ.get('NAMESPACE', 'default')
+              COLLECTOR_HOST = f"{RELEASE_NAME}-otel-collector.{NAMESPACE}.svc.cluster.local"
+              
+              logger.info(f"Waiting for Carbon receiver to be ready at {COLLECTOR_HOST}:2003...")
+              
+              # Install netcat for connectivity check
+              try:
+                  subprocess.check_call(["apk", "add", "--no-cache", "netcat-openbsd"])
+                  logger.info("Installed netcat for connectivity check")
+              except Exception as e:
+                  logger.warning(f"Could not install netcat: {e}")
+              
+              # Wait for collector to be ready
+              while True:
+                  try:
+                      result = subprocess.run(["nc", "-z", COLLECTOR_HOST, "2003"], 
+                                           capture_output=True, timeout=10)
+                      if result.returncode == 0:
+                          logger.info("Carbon receiver is ready!")
+                          break
+                      else:
+                          logger.info("Collector not ready, waiting 5 seconds...")
+                          time.sleep(5)
+                  except Exception as e:
+                      logger.info(f"Error checking collector connectivity: {e}, waiting 5 seconds...")
+                      time.sleep(5)
+              
+              logger.info("Configuring Carbon metric generator...")
+              # Configure Carbon client
+              collector_host = os.environ.get('COLLECTOR_HOST', 'logzio-telemetry-otel-collector.monitoring.svc.cluster.local')
+              default_env_id = 'carbon-env'
+              env_id = os.environ.get('ENV_ID', default_env_id)
+              
+              logger.info(f"Using collector host: {collector_host}")
+              logger.info(f"Using env_id: {env_id}")
+              
+              logger.info(f"Carbon metric generator started. Sending metrics to: {collector_host}:2003")
+              
+              # Send metrics continuously
+              while True:
+                  try:
+                      timestamp = int(time.time())
+                      
+                      # Send the Carbon metric
+                      metric_data = f"test.carbon.metric;env_id={env_id};source=carbon-gen 1 {timestamp}"
```

Comment:
This is graphite tag format, not sure if we want to test this format or dot notation format (no tags)